conditionals.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var year = prompt("¿how old are you?");
+    if (year>18){
+      alert("welcome a our page");
```

Comment:
The welcome has to be to somewhere

```suggestion
      alert("Welcome to our page");
```